keycloak/import/hmda-realm.json

```diff
@@ -205,7 +205,7 @@
   "groups" : [ ],
   "defaultRoles" : [ "uma_authorization", "offline_access" ],
   "requiredCredentials" : [ "password" ],
-  "passwordPolicy" : "hashIterations(20000)",
+  "passwordPolicy" : "hashIterations and forceExpiredPasswordChange(30) and passwordHistory(12) and upperCase and lowerCase and digits and specialChars and length(12) and notUsername",
```

Comment:
Seems reasonable to me.  I'll be talking to security folks in the coming days.  I'll find out what our options are.